scripts/check-users-in-space-developer-role.ps1

```diff
@@ -1,37 +1,81 @@
 #!/usr/bin/env pwsh
 
-param([String] $space, [String] $slackwebhook)
-
-$data = @()
+param($space_name, $slackwebhook, $delete_arg, $protected_usernames_csv)
```

Comment:
param([Parameter(Mandatory)] $space_name, $slackwebhook, $delete_arg, $protected_usernames_csv) ........?

Reply:
param($space_name, $slackwebhook, $delete_arg, $protected_usernames_csv='twd-bat-paas-prod@digital.education.gov.uk')

Reply:
Added [Parameter(Mandatory)] thanks

We agreed to not set bat default as it's a generic script